security/oc_keypair_internal.h

```diff
@@ -0,0 +1,141 @@
+/****************************************************************************
+ *
+ * Copyright (c) 2018-2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"),
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef OC_KEYPAIR_H
+#define OC_KEYPAIR_H
+
+#if defined(OC_SECURITY) && defined(OC_PKI)
+
+#include "oc_rep.h"
+
+#include <mbedtls/ecp.h>
+#include <stdbool.h>
+#include <stddef.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define OC_ECDSA_PUBKEY_SIZE (91)
```

Comment:
Is this size enough large for p384 (pubkey_size na privkey_size) i thnik this size depends on size of curve.